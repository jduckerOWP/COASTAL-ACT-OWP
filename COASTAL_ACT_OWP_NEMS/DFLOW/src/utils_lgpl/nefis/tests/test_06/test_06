#!/bin/sh
cp ../test_05/nefis_ex.dat .
cp ../test_05/nefis_ex.def .
./test_06_dependent
