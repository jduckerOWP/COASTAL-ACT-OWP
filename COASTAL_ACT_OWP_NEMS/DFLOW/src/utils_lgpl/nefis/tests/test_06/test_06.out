Flat profile:

Each sample counts as 0.01 seconds.
  %   cumulative   self              self     total           
 time   seconds   seconds    calls   s/call   s/call  name    
 36.51      2.11     2.11    80931     0.00     0.00  convert_ieee
 11.25      2.76     0.65     6728     0.00     0.00  GP_variable_pointer
 10.38      3.36     0.60    26888     0.00     0.00  Put_element
  9.69      3.92     0.56                             _mcount
  9.34      4.46     0.54    26892     0.00     0.00  Get_element
  4.33      4.71     0.25        1     0.25     4.93  MAIN__
  2.77      4.87     0.16   188363     0.00     0.00  BUInt4Fseek
  2.25      5.00     0.13                             mcount
  2.08      5.12     0.12   107577     0.00     0.00  F_Copy_text
  1.90      5.23     0.11    80658     0.00     0.00  GP_write_file
  1.56      5.32     0.09   107705     0.00     0.00  GP_read_file
  1.56      5.41     0.09    53780     0.00     0.00  RT_retrieve_struc
  1.21      5.48     0.07                             lseek
  0.87      5.53     0.05    26888     0.00     0.00  putels_
  0.69      5.57     0.04   269068     0.00     0.00  RT_streql
  0.69      5.61     0.04    53780     0.00     0.00  RT_retrieve
  0.69      5.65     0.04    26892     0.00     0.00  getelt_
  0.52      5.68     0.03    53780     0.00     0.00  Find
  0.35      5.70     0.02                             free
  0.35      5.72     0.02                             write
  0.17      5.73     0.01    26888     0.00     0.00  putelt_
  0.17      5.74     0.01                             e_wsfe
  0.17      5.75     0.01                             malloc
  0.17      5.76     0.01                             read
  0.17      5.77     0.01                             strncmp
  0.17      5.78     0.01                             strncpy
  0.00      5.78     0.00     6729     0.00     0.00  RT_retrieve_var
  0.00      5.78     0.00       75     0.00     0.00  HS_check_ecg
  0.00      5.78     0.00       75     0.00     0.00  HS_hash_key
  0.00      5.78     0.00       57     0.00     0.00  clock_
  0.00      5.78     0.00       18     0.00     0.00  HS_get_cont_elm
  0.00      5.78     0.00       16     0.00     0.00  GP_inquire_elm
  0.00      5.78     0.00       16     0.00     0.00  HS_get_cont_cel
  0.00      5.78     0.00       16     0.00     0.00  HS_get_cont_grp
  0.00      5.78     0.00       12     0.00     0.00  GP_inquire_cel
  0.00      5.78     0.00       12     0.00     0.00  GP_inquire_dat
  0.00      5.78     0.00       12     0.00     0.00  GP_inquire_grp_def
  0.00      5.78     0.00       12     0.00     0.00  HS_get_cont_dat
  0.00      5.78     0.00       12     0.00     0.00  RT_retrieve_file
  0.00      5.78     0.00       12     0.00     0.00  RT_update_struc
  0.00      5.78     0.00       12     0.00     0.00  Tree
  0.00      5.78     0.00        4     0.00     0.00  Define_data
  0.00      5.78     0.00        4     0.00     0.00  Define_group
  0.00      5.78     0.00        4     0.00     0.00  credat_
  0.00      5.78     0.00        4     0.00     0.00  defgrp_
  0.00      5.78     0.00        2     0.00     0.00  GP_flush_hash
  0.00      5.78     0.00        2     0.00     0.00  RemoveTrailingBlanks
  0.00      5.78     0.00        2     0.00     0.00  close_nefis_files
  0.00      5.78     0.00        2     0.00     0.00  create_nefis_files
  0.00      5.78     0.00        2     0.00     0.00  open_nefis_file
  0.00      5.78     0.00        1     0.00     0.00  Define_cel
  0.00      5.78     0.00        1     0.00     0.00  RT_update_var_array
  0.00      5.78     0.00        1     0.00     0.00  RT_update_var_index_array
  0.00      5.78     0.00        1     0.00     0.00  clsdat_
  0.00      5.78     0.00        1     0.00     0.00  clsdef_
  0.00      5.78     0.00        1     0.00     0.00  defcel_
  0.00      5.78     0.00        1     0.00     0.00  neferr_
  0.00      5.78     0.00        1     0.00     0.00  nefis_error
  0.00      5.78     0.00        1     0.00     0.00  opndat_
  0.00      5.78     0.00        1     0.00     0.00  opndef_

 %         the percentage of the total running time of the
time       program used by this function.

cumulative a running sum of the number of seconds accounted
 seconds   for by this function and those listed above it.

 self      the number of seconds accounted for by this
seconds    function alone.  This is the major sort for this
           listing.

calls      the number of times this function was invoked, if
           this function is profiled, else blank.
 
 self      the average number of milliseconds spent in this
ms/call    function per call, if this function is profiled,
	   else blank.

 total     the average number of milliseconds spent in this
ms/call    function and its descendents per call, if this 
	   function is profiled, else blank.

name       the name of the function.  This is the minor sort
           for this listing. The index shows the location of
	   the function in the gprof listing. If the index is
	   in parenthesis it shows where it would appear in
	   the gprof listing if it were to be printed.

		     Call graph (explanation follows)


granularity: each sample hit covers 4 byte(s) for 0.20% of 5.09 seconds

index % time    self  children    called     name
                0.25    4.68       1/1           main [2]
[1]     96.9    0.25    4.68       1         MAIN__ [1]
                0.01    3.09   26888/26888       putelt_ [3]
                0.04    1.54   26892/26892       getelt_ [7]
                0.00    0.00       4/4           credat_ [32]
                0.00    0.00       4/4           defgrp_ [37]
                0.00    0.00       1/1           defcel_ [40]
                0.00    0.00       1/1           opndef_ [49]
                0.00    0.00       1/1           opndat_ [48]
                0.00    0.00       1/1           clsdat_ [50]
                0.00    0.00       1/1           clsdef_ [51]
                0.00    0.00      57/57          clock_ [57]
                0.00    0.00       1/1           neferr_ [62]
-----------------------------------------------
                                                 <spontaneous>
[2]     96.9    0.00    4.93                 main [2]
                0.25    4.68       1/1           MAIN__ [1]
-----------------------------------------------
                0.01    3.09   26888/26888       MAIN__ [1]
[3]     60.9    0.01    3.09   26888         putelt_ [3]
                0.05    3.04   26888/26888       putels_ [4]
-----------------------------------------------
                0.05    3.04   26888/26888       putelt_ [3]
[4]     60.7    0.05    3.04   26888         putels_ [4]
                0.60    2.38   26888/26888       Put_element [5]
                0.06    0.00   53776/107577      F_Copy_text [15]
-----------------------------------------------
                0.60    2.38   26888/26888       putels_ [4]
[5]     58.5    0.60    2.38   26888         Put_element [5]
                0.65    0.75    6728/6728        GP_variable_pointer [9]
                0.70    0.00   26888/80931       convert_ieee [6]
                0.11    0.07   80640/80658       GP_write_file [12]
                0.02    0.08   26888/53780       RT_retrieve [10]
-----------------------------------------------
                0.00    0.00       2/80931       create_nefis_files [45]
                0.00    0.00       2/80931       GP_flush_hash [46]
                0.00    0.00       2/80931       Define_cel [41]
                0.00    0.00       4/80931       RT_update_var_index_array [44]
                0.00    0.00       5/80931       Define_data [33]
                0.00    0.00       8/80931       Define_group [38]
                0.00    0.00      12/80931       HS_get_cont_dat [39]
                0.00    0.00      32/80931       HS_get_cont_grp [35]
                0.00    0.00      48/80931       HS_get_cont_cel [30]
                0.00    0.00      54/80931       HS_get_cont_elm [29]
                0.00    0.00      66/80931       HS_check_ecg [27]
                0.70    0.00   26888/80931       Put_element [5]
                0.70    0.00   26892/80931       Get_element [8]
                0.70    0.00   26916/80931       GP_variable_pointer [9]
[6]     41.5    2.11    0.00   80931         convert_ieee [6]
-----------------------------------------------
                0.04    1.54   26892/26892       MAIN__ [1]
[7]     31.0    0.04    1.54   26892         getelt_ [7]
                0.54    0.94   26892/26892       Get_element [8]
                0.06    0.00   53784/107577      F_Copy_text [15]
-----------------------------------------------
                0.54    0.94   26892/26892       getelt_ [7]
[8]     29.1    0.54    0.94   26892         Get_element [8]
                0.70    0.00   26892/80931       convert_ieee [6]
                0.07    0.07   80644/107705      GP_read_file [11]
                0.02    0.08   26892/53780       RT_retrieve [10]
                0.00    0.00    6729/6729        RT_retrieve_var [42]
-----------------------------------------------
                0.65    0.75    6728/6728        Put_element [5]
[9]     27.4    0.65    0.75    6728         GP_variable_pointer [9]
                0.70    0.00   26916/80931       convert_ieee [6]
                0.02    0.02   26909/107705      GP_read_file [11]
                0.00    0.00       7/80658       GP_write_file [12]
-----------------------------------------------
                0.02    0.08   26888/53780       Put_element [5]
                0.02    0.08   26892/53780       Get_element [8]
[10]     4.0    0.04    0.16   53780         RT_retrieve [10]
                0.09    0.07   53780/53780       RT_retrieve_struc [14]
                0.00    0.00      12/12          RT_retrieve_file [26]
                0.00    0.00      12/12          RT_update_struc [58]
-----------------------------------------------
                0.00    0.00       4/107705      create_nefis_files [45]
                0.00    0.00       4/107705      RT_update_var_index_array [44]
                0.00    0.00      12/107705      HS_get_cont_dat [39]
                0.00    0.00      16/107705      HS_get_cont_grp [35]
                0.00    0.00      18/107705      HS_get_cont_elm [29]
                0.00    0.00      32/107705      HS_get_cont_cel [30]
                0.00    0.00      66/107705      HS_check_ecg [27]
                0.02    0.02   26909/107705      GP_variable_pointer [9]
                0.07    0.07   80644/107705      Get_element [8]
[11]     3.6    0.09    0.09  107705         GP_read_file [11]
                0.09    0.00  107705/188363      BUInt4Fseek [13]
-----------------------------------------------
                0.00    0.00       1/80658       Define_cel [41]
                0.00    0.00       2/80658       GP_flush_hash [46]
                0.00    0.00       4/80658       Define_group [38]
                0.00    0.00       4/80658       Define_data [33]
                0.00    0.00       7/80658       GP_variable_pointer [9]
                0.11    0.07   80640/80658       Put_element [5]
[12]     3.5    0.11    0.07   80658         GP_write_file [12]
                0.07    0.00   80658/188363      BUInt4Fseek [13]
-----------------------------------------------
                0.07    0.00   80658/188363      GP_write_file [12]
                0.09    0.00  107705/188363      GP_read_file [11]
[13]     3.1    0.16    0.00  188363         BUInt4Fseek [13]
-----------------------------------------------
                0.09    0.07   53780/53780       RT_retrieve [10]
[14]     3.1    0.09    0.07   53780         RT_retrieve_struc [14]
                0.03    0.04   53780/53780       Find [16]
-----------------------------------------------
                0.00    0.00       1/107577      defcel_ [40]
                0.00    0.00       8/107577      credat_ [32]
                0.00    0.00       8/107577      defgrp_ [37]
                0.06    0.00   53776/107577      putels_ [4]
                0.06    0.00   53784/107577      getelt_ [7]
[15]     2.4    0.12    0.00  107577         F_Copy_text [15]
-----------------------------------------------
                0.03    0.04   53780/53780       RT_retrieve_struc [14]
[16]     1.4    0.03    0.04   53780         Find [16]
                0.04    0.00  269068/269068      RT_streql [18]
-----------------------------------------------
                                                 <spontaneous>
[17]     1.4    0.07    0.00                 lseek [17]
-----------------------------------------------
                0.04    0.00  269068/269068      Find [16]
[18]     0.8    0.04    0.00  269068         RT_streql [18]
-----------------------------------------------
                                                 <spontaneous>
[19]     0.4    0.02    0.00                 free [19]
-----------------------------------------------
                                                 <spontaneous>
[20]     0.4    0.02    0.00                 write [20]
-----------------------------------------------
                                                 <spontaneous>
[21]     0.2    0.01    0.00                 e_wsfe [21]
-----------------------------------------------
                                                 <spontaneous>
[22]     0.2    0.01    0.00                 malloc [22]
-----------------------------------------------
                                                 <spontaneous>
[23]     0.2    0.01    0.00                 read [23]
-----------------------------------------------
                                                 <spontaneous>
[24]     0.2    0.01    0.00                 strncmp [24]
-----------------------------------------------
                                                 <spontaneous>
[25]     0.2    0.01    0.00                 strncpy [25]
-----------------------------------------------
                0.00    0.00      12/12          RT_retrieve [10]
[26]     0.1    0.00    0.00      12         RT_retrieve_file [26]
                0.00    0.00      16/16          GP_inquire_elm [28]
                0.00    0.00      12/12          GP_inquire_cel [31]
                0.00    0.00      12/12          GP_inquire_grp_def [34]
                0.00    0.00      12/12          GP_inquire_dat [36]
-----------------------------------------------
                0.00    0.00       3/75          Define_cel [41]
                0.00    0.00       8/75          Define_group [38]
                0.00    0.00      12/75          GP_inquire_cel [31]
                0.00    0.00      12/75          GP_inquire_dat [36]
                0.00    0.00      12/75          GP_inquire_grp_def [34]
                0.00    0.00      12/75          Define_data [33]
                0.00    0.00      16/75          GP_inquire_elm [28]
[27]     0.0    0.00    0.00      75         HS_check_ecg [27]
                0.00    0.00      66/80931       convert_ieee [6]
                0.00    0.00      66/107705      GP_read_file [11]
                0.00    0.00      75/75          HS_hash_key [56]
-----------------------------------------------
                0.00    0.00      16/16          RT_retrieve_file [26]
[28]     0.0    0.00    0.00      16         GP_inquire_elm [28]
                0.00    0.00      16/18          HS_get_cont_elm [29]
                0.00    0.00      16/75          HS_check_ecg [27]
-----------------------------------------------
                0.00    0.00       2/18          Define_cel [41]
                0.00    0.00      16/18          GP_inquire_elm [28]
[29]     0.0    0.00    0.00      18         HS_get_cont_elm [29]
                0.00    0.00      54/80931       convert_ieee [6]
                0.00    0.00      18/107705      GP_read_file [11]
-----------------------------------------------
                0.00    0.00       4/16          Define_data [33]
                0.00    0.00      12/16          GP_inquire_cel [31]
[30]     0.0    0.00    0.00      16         HS_get_cont_cel [30]
                0.00    0.00      48/80931       convert_ieee [6]
                0.00    0.00      32/107705      GP_read_file [11]
-----------------------------------------------
                0.00    0.00      12/12          RT_retrieve_file [26]
[31]     0.0    0.00    0.00      12         GP_inquire_cel [31]
                0.00    0.00      12/16          HS_get_cont_cel [30]
                0.00    0.00      12/75          HS_check_ecg [27]
-----------------------------------------------
                0.00    0.00       4/4           MAIN__ [1]
[32]     0.0    0.00    0.00       4         credat_ [32]
                0.00    0.00       4/4           Define_data [33]
                0.00    0.00       8/107577      F_Copy_text [15]
-----------------------------------------------
                0.00    0.00       4/4           credat_ [32]
[33]     0.0    0.00    0.00       4         Define_data [33]
                0.00    0.00       4/16          HS_get_cont_cel [30]
                0.00    0.00      12/75          HS_check_ecg [27]
                0.00    0.00       4/16          HS_get_cont_grp [35]
                0.00    0.00       5/80931       convert_ieee [6]
                0.00    0.00       4/80658       GP_write_file [12]
-----------------------------------------------
                0.00    0.00      12/12          RT_retrieve_file [26]
[34]     0.0    0.00    0.00      12         GP_inquire_grp_def [34]
                0.00    0.00      12/16          HS_get_cont_grp [35]
                0.00    0.00      12/75          HS_check_ecg [27]
-----------------------------------------------
                0.00    0.00       4/16          Define_data [33]
                0.00    0.00      12/16          GP_inquire_grp_def [34]
[35]     0.0    0.00    0.00      16         HS_get_cont_grp [35]
                0.00    0.00      32/80931       convert_ieee [6]
                0.00    0.00      16/107705      GP_read_file [11]
-----------------------------------------------
                0.00    0.00      12/12          RT_retrieve_file [26]
[36]     0.0    0.00    0.00      12         GP_inquire_dat [36]
                0.00    0.00      12/12          HS_get_cont_dat [39]
                0.00    0.00      12/75          HS_check_ecg [27]
-----------------------------------------------
                0.00    0.00       4/4           MAIN__ [1]
[37]     0.0    0.00    0.00       4         defgrp_ [37]
                0.00    0.00       4/4           Define_group [38]
                0.00    0.00       8/107577      F_Copy_text [15]
-----------------------------------------------
                0.00    0.00       4/4           defgrp_ [37]
[38]     0.0    0.00    0.00       4         Define_group [38]
                0.00    0.00       8/80931       convert_ieee [6]
                0.00    0.00       8/75          HS_check_ecg [27]
                0.00    0.00       4/80658       GP_write_file [12]
-----------------------------------------------
                0.00    0.00      12/12          GP_inquire_dat [36]
[39]     0.0    0.00    0.00      12         HS_get_cont_dat [39]
                0.00    0.00      12/80931       convert_ieee [6]
                0.00    0.00      12/107705      GP_read_file [11]
-----------------------------------------------
                0.00    0.00       1/1           MAIN__ [1]
[40]     0.0    0.00    0.00       1         defcel_ [40]
                0.00    0.00       1/1           Define_cel [41]
                0.00    0.00       1/107577      F_Copy_text [15]
-----------------------------------------------
                0.00    0.00       1/1           defcel_ [40]
[41]     0.0    0.00    0.00       1         Define_cel [41]
                0.00    0.00       2/18          HS_get_cont_elm [29]
                0.00    0.00       3/75          HS_check_ecg [27]
                0.00    0.00       2/80931       convert_ieee [6]
                0.00    0.00       1/80658       GP_write_file [12]
-----------------------------------------------
                0.00    0.00    6729/6729        Get_element [8]
[42]     0.0    0.00    0.00    6729         RT_retrieve_var [42]
                0.00    0.00       1/1           RT_update_var_array [43]
-----------------------------------------------
                0.00    0.00       1/1           RT_retrieve_var [42]
[43]     0.0    0.00    0.00       1         RT_update_var_array [43]
                0.00    0.00       1/1           RT_update_var_index_array [44]
-----------------------------------------------
                                   3             RT_update_var_index_array [44]
                0.00    0.00       1/1           RT_update_var_array [43]
[44]     0.0    0.00    0.00       1+3       RT_update_var_index_array [44]
                0.00    0.00       4/80931       convert_ieee [6]
                0.00    0.00       4/107705      GP_read_file [11]
                                   3             RT_update_var_index_array [44]
-----------------------------------------------
                0.00    0.00       1/2           opndat_ [48]
                0.00    0.00       1/2           opndef_ [49]
[45]     0.0    0.00    0.00       2         create_nefis_files [45]
                0.00    0.00       2/80931       convert_ieee [6]
                0.00    0.00       4/107705      GP_read_file [11]
                0.00    0.00       2/2           open_nefis_file [61]
-----------------------------------------------
                0.00    0.00       2/2           close_nefis_files [47]
[46]     0.0    0.00    0.00       2         GP_flush_hash [46]
                0.00    0.00       2/80931       convert_ieee [6]
                0.00    0.00       2/80658       GP_write_file [12]
-----------------------------------------------
                0.00    0.00       1/2           clsdat_ [50]
                0.00    0.00       1/2           clsdef_ [51]
[47]     0.0    0.00    0.00       2         close_nefis_files [47]
                0.00    0.00       2/2           GP_flush_hash [46]
-----------------------------------------------
                0.00    0.00       1/1           MAIN__ [1]
[48]     0.0    0.00    0.00       1         opndat_ [48]
                0.00    0.00       1/2           create_nefis_files [45]
                0.00    0.00       1/2           RemoveTrailingBlanks [60]
-----------------------------------------------
                0.00    0.00       1/1           MAIN__ [1]
[49]     0.0    0.00    0.00       1         opndef_ [49]
                0.00    0.00       1/2           create_nefis_files [45]
                0.00    0.00       1/2           RemoveTrailingBlanks [60]
-----------------------------------------------
                0.00    0.00       1/1           MAIN__ [1]
[50]     0.0    0.00    0.00       1         clsdat_ [50]
                0.00    0.00       1/2           close_nefis_files [47]
-----------------------------------------------
                0.00    0.00       1/1           MAIN__ [1]
[51]     0.0    0.00    0.00       1         clsdef_ [51]
                0.00    0.00       1/2           close_nefis_files [47]
-----------------------------------------------
                0.00    0.00      75/75          HS_check_ecg [27]
[56]     0.0    0.00    0.00      75         HS_hash_key [56]
-----------------------------------------------
                0.00    0.00      57/57          MAIN__ [1]
[57]     0.0    0.00    0.00      57         clock_ [57]
-----------------------------------------------
                0.00    0.00      12/12          RT_retrieve [10]
[58]     0.0    0.00    0.00      12         RT_update_struc [58]
                0.00    0.00      12/12          Tree [59]
-----------------------------------------------
                                  66             Tree [59]
                0.00    0.00      12/12          RT_update_struc [58]
[59]     0.0    0.00    0.00      12+66      Tree [59]
                                  66             Tree [59]
-----------------------------------------------
                0.00    0.00       1/2           opndat_ [48]
                0.00    0.00       1/2           opndef_ [49]
[60]     0.0    0.00    0.00       2         RemoveTrailingBlanks [60]
-----------------------------------------------
                0.00    0.00       2/2           create_nefis_files [45]
[61]     0.0    0.00    0.00       2         open_nefis_file [61]
-----------------------------------------------
                0.00    0.00       1/1           MAIN__ [1]
[62]     0.0    0.00    0.00       1         neferr_ [62]
                0.00    0.00       1/1           nefis_error [63]
-----------------------------------------------
                0.00    0.00       1/1           neferr_ [62]
[63]     0.0    0.00    0.00       1         nefis_error [63]
-----------------------------------------------

 This table describes the call tree of the program, and was sorted by
 the total amount of time spent in each function and its children.

 Each entry in this table consists of several lines.  The line with the
 index number at the left hand margin lists the current function.
 The lines above it list the functions that called this function,
 and the lines below it list the functions this one called.
 This line lists:
     index	A unique number given to each element of the table.
		Index numbers are sorted numerically.
		The index number is printed next to every function name so
		it is easier to look up where the function in the table.

     % time	This is the percentage of the `total' time that was spent
		in this function and its children.  Note that due to
		different viewpoints, functions excluded by options, etc,
		these numbers will NOT add up to 100%.

     self	This is the total amount of time spent in this function.

     children	This is the total amount of time propagated into this
		function by its children.

     called	This is the number of times the function was called.
		If the function called itself recursively, the number
		only includes non-recursive calls, and is followed by
		a `+' and the number of recursive calls.

     name	The name of the current function.  The index number is
		printed after it.  If the function is a member of a
		cycle, the cycle number is printed between the
		function's name and the index number.


 For the function's parents, the fields have the following meanings:

     self	This is the amount of time that was propagated directly
		from the function into this parent.

     children	This is the amount of time that was propagated from
		the function's children into this parent.

     called	This is the number of times this parent called the
		function `/' the total number of times the function
		was called.  Recursive calls to the function are not
		included in the number after the `/'.

     name	This is the name of the parent.  The parent's index
		number is printed after it.  If the parent is a
		member of a cycle, the cycle number is printed between
		the name and the index number.

 If the parents of the function cannot be determined, the word
 `<spontaneous>' is printed in the `name' field, and all the other
 fields are blank.

 For the function's children, the fields have the following meanings:

     self	This is the amount of time that was propagated directly
		from the child into the function.

     children	This is the amount of time that was propagated from the
		child's children to the function.

     called	This is the number of times the function called
		this child `/' the total number of times the child
		was called.  Recursive calls by the child are not
		listed in the number after the `/'.

     name	This is the name of the child.  The child's index
		number is printed after it.  If the child is a
		member of a cycle, the cycle number is printed
		between the name and the index number.

 If there are any cycles (circles) in the call graph, there is an
 entry for the cycle-as-a-whole.  This entry shows who called the
 cycle (as parents) and the members of the cycle (as children.)
 The `+' recursive calls entry shows the number of function calls that
 were internal to the cycle, and the calls entry for each member shows,
 for that member, how many times it was called from other members of
 the cycle.


Index by function name

  [13] BUInt4Fseek            [56] HS_hash_key            [32] credat_
  [41] Define_cel              [1] MAIN__                 [40] defcel_
  [33] Define_data             [5] Put_element            [37] defgrp_
  [38] Define_group           [10] RT_retrieve            [21] e_wsfe
  [15] F_Copy_text            [26] RT_retrieve_file       [19] free
  [16] Find                   [14] RT_retrieve_struc       [7] getelt_
  [46] GP_flush_hash          [42] RT_retrieve_var        [17] lseek
  [31] GP_inquire_cel         [18] RT_streql              [22] malloc
  [36] GP_inquire_dat         [58] RT_update_struc       (211) mcount
  [28] GP_inquire_elm         [43] RT_update_var_array    [62] neferr_
  [34] GP_inquire_grp_def     [44] RT_update_var_index_array [63] nefis_error
  [11] GP_read_file           [60] RemoveTrailingBlanks   [61] open_nefis_file
   [9] GP_variable_pointer    [59] Tree                   [48] opndat_
  [12] GP_write_file          (99) _mcount                [49] opndef_
   [8] Get_element            [57] clock_                  [4] putels_
  [27] HS_check_ecg           [47] close_nefis_files       [3] putelt_
  [30] HS_get_cont_cel        [50] clsdat_                [23] read
  [39] HS_get_cont_dat        [51] clsdef_                [24] strncmp
  [29] HS_get_cont_elm         [6] convert_ieee           [25] strncpy
  [35] HS_get_cont_grp        [45] create_nefis_files     [20] write
